api4/handlers.go

```diff
@@ -196,3 +199,13 @@ func (api *API) APILocal(h func(*Context, http.ResponseWriter, *http.Request)) h
 	}
 	return handler
 }
+
+func requireLicense(f handlerFunc) handlerFunc {
+	return func(c *Context, w http.ResponseWriter, r *http.Request) {
+		if c.App.Srv().License() == nil {
+			c.Err = model.NewAppError("", "api.license_error", nil, "", http.StatusNotImplemented)
```

Comment:
Should the error just be a Forbidden? The server would be able to serve the request if the license was there. Same question for similar occurrences.

Reply:
Yes I think you're right, 501 Not Implemented is the wrong code and 403 Forbidden seems more appropriate.

Reply:
@streamer45 I just realized that while I agree, this is a breaking change because the existing license checks respond with StatusNotImplemented, so it'a bit of a bigger (and breaking) change.

Reply:
Yes, I agree on 2 as well. Could we just file a ticket to address this on the next API version (if appropriate)?